Jenkinsfile

```diff
@@ -22,6 +23,10 @@ pipeline {
     stage("Setup") {
       steps {
         sh """
+          if [ ! -z '${env.EMAIL}' ]; then
+            sudo yum install -y /usr/sbin/postfix
```

Comment:
Don't you have pre-prepared jenkins slaves where you should have installed all the dependency needed for running jobs? I think that this shouldn't be part of each run if you have some static jenkins slaves but should be part of some ansible playbook which you are using for setup slave or part of image if you are using containerized slave, same for line bellow.

Reply:
No - we use vanilla images, so there's no ansible playbook at all.